components/orderForm/OrderDetails.js

```diff
@@ -37,6 +40,12 @@ export default function OrderDetails({children}) {
       return
     }
 
+    if (inventoryInfo && inventoryInfo.markdown) {
+      inventoryInfo = inventoryInfo.markdown.split(/(?=####)/g)
+
+      inventoryInfo = inventoryInfo.filter(x => x !== "")
+    }
+
```

Comment:
whoops we accidentally put this block inside a different map function (the `Object.keys(items).forEach(...)` that starts on line 31)

can you move this up to maybe line 22?

Reply:
i moved it up! hope i did that right lol